### DISCUSSION TITLE
Can't set customEnv variable

### DISCUSSION THREAD

**Discussion by mantasmatulis** [Kong/kong#9017] [Help]
Hi, 
I need to set the env variable for my custom Kong plugin and I use `customEnv` property to achieve that. Unfortunately, when I set any value to that key the customEnv variable doesn't appear. I set it in the values.yaml file but the variable doesn't exist when Kong starts. Any ideas or suggestions on how to use it?
e.g.
```yaml
customEnv:
  TEST: "hello"
```

**Comment by StarlightIbuki** [comment]
By default, Nginx removes environment variables when starting workers. You could save the value of this env at `init` phase (before the fork) and use this in later phases.
Or, you could insert the `env TEST;` directive.
